### PR TITLE
Fix button selector for Lightning app e2e tests

### PR DESCRIPTION
**Pull request by alecmerdler**
## What does this PR do?

Changes to the Lightning platform UI caused the e2e tests
to break because button text on the project creation dialog
was changed. This fixes the selector.

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [x] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by awaelchli — APPROVED** [review]

File: src/lightning_app/testing/testing.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def run_app_in_cloud(app_folder: str, app_name: str = "app.py") -> Generator:
         admin_page.goto(f"{Config.url}/{Config.username}/apps")
         try:
             # Closing the Create Project modal
-            button = admin_page.locator('button:has-text("Cancel")')
```
Comment (manskx) [inline src/lightning_app/testing/testing.py:252]: To make the e2e pass, maybe we can allow both either `Cancel` or `Continue` ?

`'button:has-text("Cancel"), button:has-text("Continue")'`

Comment (alecmerdler) [reply src/lightning_app/testing/testing.py:252]: @manskx Ah yes, I was actually going to try and do this but I couldn't figure out the sytax for Python+Playwright. Let me update the code to do that instead.

**Review by hhsecond — APPROVED** [review]

**Review by manskx — APPROVED** [review]

File: src/lightning_app/testing/testing.py (hunk 1 above)
Comment (manskx) [reply src/lightning_app/testing/testing.py:245]: It will break if we have both buttons, hopefully not 😅

**Review by Borda — APPROVED** [review]

**Pull request merged**